tools/flang2/flang2exe/symacc.cpp

```diff
@@ -40,7 +40,9 @@ sym_init_first(void)
   int i;
 
   int sizeof_SYM = sizeof(SYM) / sizeof(INT);
-  assert(sizeof_SYM == 36, "bad SYM size", sizeof_SYM, ERR_Fatal);
+#ifndef _WIN64
+// assert(sizeof_SYM == 36, "bad SYM size", sizeof_SYM, 4);
```

Comment:
Is it necessary comment it out if it is guarded?

Reply:
I seem to recall that it failed the CI.

Reply:
https://github.com/flang-compiler/flang/runs/2282723391#step:10:387

Reply:
Maybe this block should be removed.

Reply:
Is it not possible to compare with a different size when building on Windows? e.g.
```c
#ifdef _WIN64
  assert(sizeof_SYM == 48, "bad SYM size", sizeof_SYM, ERR_Fatal);
#else
  assert(sizeof_SYM == 36, "bad SYM size", sizeof_SYM, ERR_Fatal);
#endif
```

Reply:
Why have you changed from ERR_Fatal to its value 4? If you keep it as ERR_Fatal wouldn't it work?

The patch looks OK to me.

Reply:
64bit Windows uses the LLP64 data model where i.e. long is still 32bit. SYM size as fas as I remember should be 44. These sym size comes from legacy_ints definitons, am I right?

Reply:
@xoviat a reply to @kaadam's and @bryanpkc's point here will be great.

Reply:
ping @xoviat